cdap-kafka/src/main/java/co/cask/cdap/kafka/run/KafkaServerMain.java

```diff
@@ -55,10 +57,11 @@ public static void main(String [] args) throws Exception {
   public void init(String[] args) {
     CConfiguration cConf = CConfiguration.create();
     String zkConnectStr = cConf.get(Constants.Zookeeper.QUORUM);
-    String zkNamespace = cConf.get(KafkaConstants.ConfigKeys.ZOOKEEPER_NAMESPACE_CONFIG);
+    String zkNamespace = cConf.get(KafkaConstants.ConfigKeys.ZOOKEEPER_NAMESPACE_CONFIG_DEPRECATED);
+
+    int port = cConf.getInt(KafkaConstants.ConfigKeys.PORT_CONFIG_DEPRECATED, -1);
```

Comment:
but it seems like we want to check that the port is valid (not -1), which I guess we can do in the for loop, but it also makes sense like this IMO.